_includes/scenarios/use/delete-identity-from-enmeshed.md

```diff
@@ -0,0 +1,13 @@
+{% include warnings/feature-work-in-progress %}
+
+# Flow
+
+Every Identity has the option of deleting its Identity from the Backbone. There are three options to delete the Identity:
```

Comment:
this sounds strange. wtb `Every Identity has the option of deleting itself from the Backbone.`